photoeditor/src/main/java/ja/burhanrashid52/photoeditor/TextStyleBuilder.java

```diff
@@ -145,6 +164,10 @@ protected void applyTextSize(TextView textView, float size) {
         textView.setTextSize(size);
     }
 
+    protected void applyTextShadow(TextView textView, float radius, float dx, float dy, int color) {
```

Comment:
You can replace this method with [this](https://github.com/burhanrashid52/PhotoEditor/blob/master/photoeditor/src/main/java/ja/burhanrashid52/photoeditor/TextStyleBuilder.java#L227). And just past TextShadow from in line 118.

You can also remove `TextShadow` inside the `TextShadowBuilder`

You need update with the latest master to this method.